### PR TITLE
Add changelog and Download section to readme.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,14 @@ java -jar composer-latest-version.jar \
 --verbose-output false
 ```
 
+### Download
+
+Composer [published to jcenter](https://jcenter.bintray.com/com/gojuno/composer) and follows [Semantic Versioning](http://semver.org/).
```
Comment (ming13) [inline README.md]: `Composer is available at jcenter`. Versioning note is kind of useless in most cases.

File: README.md [hunk 2]
```diff
@@ -81,6 +81,14 @@ java -jar composer-latest-version.jar \
...
+
+You can download it in your CI scripts or store it in your version control system (not recommended).
+
+All the releases and changelogs can be found on [Releases Page](https://github.com/gojuno/composer/releases).
```
Comment (ming13) [inline README.md]: Maven dependency block?

Comment (artem-zinnatullin) [inline README.md]: Well, I'm not sure it's useful as library. Gradle Plugin would be better.

Comment (ming13) [inline README.md]: Agree, but for now it is still possible to write some sort of custom `composer-launcher` and execute it instead of hosting fat JAR.

Comment (artem-zinnatullin) [inline README.md]: nniiiiiiiice idea, we can create one in pure java ourselves and proguard it to minify binary size to less than 100kb

**Pull request merged**